testing/compose/stack-trader.yml

```diff
@@ -189,3 +189,20 @@ services:
       - "NATS_URL=nats://nats:4222"
       - "REDIS_URL=redis://redis:6379"
       - "NATS_SUBSCRIPTION=decs.frames.*.mining, decs.system.registry"
+  merchant:
+    image: stacktrader/merchant
+    expose:
+      - "9009"
+    ports:
+      - "9009:9009"
```

Comment:
You're not setting the PORT environment variable here, so there's nothing saying it'll be on port 9009, it will default to 9000

Reply:
This one is for the docker-compose, right? We won't use this for the cluster, but it seems like all of our systems locally will go to port 9000